src/components/standalone/firewall/conntrack/ConntrackContent.vue

```diff
@@ -0,0 +1,123 @@
+}
+
+async function fetchConntrack(): Promise<ConntrackRecord[]> {
+  const ConntrackRecordsResponse = await ubusCall('ns.conntrack', 'list')
+  // Function to update source and destination properties
+  function updateConntrackRecords(records: ConntrackRecord[]) {
+    return records.map((record) => {
+      // ipv6 localhost is ::1, translate it to ipv4
+      if (record.destination == '::1') {
+        record.destination = '127.0.0.1'
+      }
+      if (record.source == '::1') {
+        record.source = '127.0.0.1'
+      }
+      // translate the source and destination with the port
+      if (record.source_port) {
+        record.source = record.source + ' : ' + record.source_port
+      }
+      if (record.destination_port) {
+        record.destination = record.destination + ' : ' + record.destination_port
+      }
+      // translate the timeout
+      if (record.timeout) {
+        record.timeout = record.timeout + ' s'
+      }
+      // translate statistics download and upload
+      record.statistics =
+        byteFormat1024(record.source_stats.bytes) +
+        ' / ' +
+        byteFormat1024(record.destination_stats.bytes)
+      return record
+    })
+  }
+  ConntrackRecordsResponse.data = updateConntrackRecords(ConntrackRecordsResponse.data.data)
+
+  return ConntrackRecordsResponse.data
+}
+
+function applyFilterToConntrackRecords(records: ConntrackRecord[], filter: string) {
+  return records.filter(
+    (ConntrackRecord) =>
+      ConntrackRecord.source.includes(filter) ||
+      ConntrackRecord.destination.includes(filter) ||
+      ConntrackRecord.protocol.includes(filter) ||
+      (ConntrackRecord.state ?? '').includes(filter) ||
+      (ConntrackRecord.source_port ?? '').includes(filter) ||
+      (ConntrackRecord.destination_port ?? '').includes(filter)
+  )
+}
+
+async function getUciChanges() {
+  await uciChangesStore.getChanges()
+}
+</script>
+
+<template>
+  <NeHeading tag="h3" class="mb-7">{{ t('standalone.conntrack.title') }}</NeHeading>
+  <FilterableListItemConntrack
+    :fetch-items-function="fetchConntrack"
+    :fetch-error-notification-title="t('error.cannot_retrieve_conntrack_records')"
+    :apply-filter-to-items-function="applyFilterToConntrackRecords"
+    :no-items-found-message="t('standalone.conntrack.no_connection_found')"
+    :no-filtered-items-found-message="t('standalone.conntrack.no_connection_found')"
+    :no-filtered-items-found-description="t('standalone.conntrack.filter_change_suggestion')"
+    @reload-items="getUciChanges()"
+  >
+    <template #item-list-view="{ items, openDeleteModal }">
+      <ConntrackRecordsTable :conntrack-records="items" @record-delete="openDeleteModal" />
+    </template>
+    <template #delete-item-modal="{ isModalShown, closeModal, itemToDelete, reloadItems }">
+      <DeleteConntrackRecordModal
+        :visible="isModalShown"
+        @close="closeModal()"
+        @record-deleted="reloadItems()"
+        :item-to-delete="itemToDelete"
```

Comment:
`itemToDelete` should suffice as a trigger for the modal

Reply:
done tk